.taskcluster.yml

```diff
@@ -115,6 +115,80 @@ tasks:
             owner: ${user}@users.noreply.github.com
             source: ${repository}/raw/${head_rev}/.taskcluster.yml
 ####################################################################################################
+# Task: Snapshot release
+####################################################################################################
+  - $if: 'tasks_for == "cron"'
+    then:
+      $let:
+        decision_task_id: {$eval: as_slugid("decision_task")}
+        expires_in: {$fromNow: '1 year'}
+        scheduler_id: focus-nightly-sched
```

Comment:
I like having something more generic with `mobile` keyboard within, rather than dedicated products like `focus` or such. Let's do this in a follow-up, seems like a good idea indeed 👍